Added support for using HTML inside validation error messages (#1094)
digitallyinduced/ihp: Guide/validation.markdown

This record now has a validation error attached to its title.

#### Attaching Errors with HTML

If you try to use HTML code within `attachFailure`, the HTML code will be escaped and not rendered as expected:

```haskell
post
    |> attachFailure #title "Invalid value. <a href="https://example.com/docs">Check the documentation</a>"

>     -- Link will not be clickable as it's going through escaping

Reviewer: ```suggestion
    -- Link will not be clickable as the HTML is escaped
```
